main.tf

```diff
@@ -0,0 +1,59 @@
+resource "google_iam_workload_identity_pool" "rad_security_gcr_identity_pool" {
```

Comment:
In the other RAD Connect module we create a RAD Workload Identity Pool. Could we provide an input variable for the `workload_identity_pool_id` of the other Workload Identity Pool, and only create this Identity Pool if the input variable is not set?